examples/BoxPlot.elm

```diff
@@ -0,0 +1,312 @@
+
+        whiskerBottomY =
+            yConvert whiskerBottom
+
+        indxXLeft =
+            Scale.convert scale indx
+
+        boxWidth =
+            Scale.bandwidth scale
+
+        whiskerWidth =
+            boxWidth / 3
+
+        boxMidX =
+            indxXLeft + boxWidth / 2
+
+        outliers =
+            List.Extra.takeWhile (\y -> y < whiskerBottomMin) sortedYS
+                ++ List.Extra.takeWhile (\y -> y > whiskerTopMax) revSortYS
+
+        outliersY =
+            List.map (\y -> yConvert y) outliers
+
+        outliersXY =
+            List.map (\y -> ( boxMidX, y )) outliersY
+    in
+    g [ class [ "column" ] ]
+        ([ line
+            --top vertical line through rectangle
+            [ x1 boxMidX
+            , y1 thirdQuantileY
+            , x2 boxMidX
+            , y2 whiskerTopY
+            , strokeWidth 1
+            , stroke <| Paint <| Color.black
+            , opacity <| Opacity 0.7
+            ]
+            []
+         , line
+            --bottom vertical line through rectangle
+            [ x1 boxMidX
+            , y1 whiskerBottomY
+            , x2 boxMidX
+            , y2 firstQuantileY
+            , strokeWidth 1
+            , stroke <| Paint <| Color.black
+            , opacity <| Opacity 0.7
+            ]
+            []
+         , line
+            --bottom whsiker
+            [ x1 <| boxMidX - whiskerWidth / 2
+            , y1 whiskerBottomY
+            , x2 <| boxMidX + whiskerWidth / 2
+            , y2 whiskerBottomY
+            , strokeWidth 1
+            , stroke <| Paint <| Color.black
+            ]
+            []
+         , line
+            --top whisker
+            [ x1 <| boxMidX - whiskerWidth / 2
+            , y1 whiskerTopY
+            , x2 <| boxMidX + whiskerWidth / 2
+            , y2 whiskerTopY
+            , strokeWidth 1
+            , stroke <| Paint <| Color.black
+            ]
+            []
+         , rect
+            [ x indxXLeft
+            , y thirdQuantileY
+            , width boxWidth
+            , height (firstQuantileY - thirdQuantileY)
+            , fill <| Reference "linGradientDuo"
+            , opacity <| Opacity 0.8
+            ]
+            []
+         , line
+            -- median line in middle of rectangle
+            [ x1 indxXLeft
+            , y1 serieMedianY
+            , x2 <| indxXLeft + boxWidth
+            , y2 serieMedianY
+            , strokeWidth 1
+            , stroke <| Paint <| Color.black
+            , opacity <| Opacity 0.6
+            ]
+            []
+         ]
+            ++ List.map outlierCircle outliersXY
+        )
+
+
+outlierCircle : ( Float, Float ) -> Svg msg
+outlierCircle ( x, y ) =
+    circle
+        [ cx x
+        , cy y
+        , r 3
+        , fill <| Paint <| Color.rgb255 180 20 20
+        , strokeWidth 0
+        , stroke <| PaintNone
+        , opacity <| Opacity 0.7
+        ]
+        []
+
+
+yGridLine : Int -> Float -> Svg msg
```

Comment:
Grid lines are done perhaps a bit easier with `Axis.tickSizeInner -xAxisWidth` for the yAxis. The exact style can then be customised with a bit of CSS.